distribution/services/utils/distributionutils.go

```diff
@@ -31,42 +31,73 @@ func NewReleaseBundleParams(name, version string) ReleaseBundleParams {
 	}
 }
 
-func CreateBundleBody(createBundleParams ReleaseBundleParams, dryRun bool) (*ReleaseBundleBody, error) {
+func CreateBundleBody(releaseBundleParams ReleaseBundleParams, dryRun bool) (*ReleaseBundleBody, error) {
 	var bundleQueries []BundleQuery
 	// Create release bundle queries
-	for _, specFile := range createBundleParams.SpecFiles {
-		if specFile.GetSpecType() != utils.AQL {
-			query, err := utils.CreateAqlBodyForSpecWithPattern(specFile)
-			if err != nil {
-				return nil, err
-			}
-			specFile.Aql = utils.Aql{ItemsFind: query}
+	for _, specFile := range releaseBundleParams.SpecFiles {
+		// Create AQL
+		aql, err := createAql(specFile)
+		if err != nil {
+			return nil, err
 		}
-		aql := utils.BuildQueryFromSpecFile(specFile, utils.NONE)
-		bundleQueries = append(bundleQueries, BundleQuery{Aql: aql})
+
+		// Create added properties
+		addProps, err := createAddProps(specFile)
```

Comment:
createAddProps --> createTargetProps

Reply:
Changed to createAddedProps to be consisted with Distribution REST API